packages/react-inbox/src/components/Messages/EndOfTheRoad.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import EndPagination from "../../assets/pagination_end.svg";
+import { Line } from "./styled";
+
+function EndOfTheRoad() {
+  return (
+    <div style={{
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      flex: "1 0",
+    }}>
+      <Line />
+      <EndPagination /> End Of The Road
```

Comment:
do we see someone calling this `End Of The Road` something else?

Reply:
Possibly, added title as prop